src/navigator/NavigatorWrapper.tsx

```diff
@@ -70,6 +72,12 @@ export const NavigatorWrapper = () => {
   const isInviteModalVisible = useTypedSelector((state) => state.app.inviteModalVisible)
   const routeNameRef = React.useRef()
   const inSanctionedCountry = useTypedSelector(userInSanctionedCountrySelector)
+  Logger.info(
+    'NavigatorWrapper',
+    'Statsig exp layer: isLoading',
+    useLayer('name_and_picture_screen').isLoading
```

Comment:
you're calling useLayer outside the context of the StatsigProvider, so this wont work